src/dms/core/EdFi.DataManagementService.Core.External/Frontend/FrontendRequest.cs

```diff
@@ -31,9 +31,5 @@ public record FrontendRequest(
     /// <summary>
     /// A request identifier provided by the frontend service, used for log tracing
     /// </summary>
-    TraceId TraceId,
-    /// <summary>
-    /// ApiDetails retrieved from the token, used for resource authorization
-    /// </summary>
-    ClientAuthorizations ClientAuthorizations
```

Comment:
No longer coming from the frontend. Now comes via the raw Authentication header.